content/src/cloud/features/cloud-backup-delete.md

```diff
@@ -11,7 +11,7 @@ zendesk:
 - You have administrator rights in Home Assistant
 - You have [enabled Home Assistant Cloud backups](/hc/en-us/articles/26294320337181/)
 
-## Deleting Home Assistant Cloud backups
+## To delete Home Assistant Cloud backups
```

Comment:
Cloud backups can also be deleted from the account page.

Reply:
this has already been fixed in a later PR
https://support.nabucasa.com/hc/en-us/articles/26294412868381-Deleting-a-backup-from-Home-Assistant-Cloud

![image](https://github.com/user-attachments/assets/1f2c191d-2d21-42d1-b704-75aefe238088)

it now just points you to the main docs

Reply:
That page does not describe how to delete them from the account page?

Reply:
it does. see option 2. Too hidden?
![image](https://github.com/user-attachments/assets/f9fc5ca9-3f3f-4775-9b62-5fcd72515dd0)

Reply:
oh, wait...

Reply:
fixed in the main docs. <https://github.com/home-assistant/home-assistant.io/pull/38358>